doc/ale.txt

```diff
@@ -1938,6 +1938,22 @@ g:ale_root                                                         *g:ale_root*
   LSP linter, it will not run.
 
 
+g:ale_send_to_neovim_diagnostics             *g:ale_send_to_neovim_diagnostics*
+
+  Type: Number
+  Default: 0
```

Comment:
```suggestion
  Type: |Number|
  Default: `0`
```

I prefer this style of formatting. I notice that some other documentation entries like the above one aren't quite right either.